providers/transip/transipProvider.go

```diff
@@ -0,0 +1,194 @@
+package transip
+
```

Comment:
Please add a comment with a little into about transip (i.e. the URL, the company name, etc.).